IPython/terminal/shortcuts.py

```diff
@@ -47,6 +48,12 @@ def create_ipython_shortcuts(shell):
                             & insert_mode
                         ))(return_handler)
 
+    try:
+        g = parso.load_grammar()
```

Comment:
it looks like this would load every time this module gets imported: it isn't needed until a keyboard shortcut is actually tested. would it be possible to do this in a lazy fashion?